docs/RELEASE-NOTES.rst

```diff
@@ -6,15 +6,17 @@ next release
 Added Functionality
 `````````````````````
 * Changed container base image from debian-stretch to debian-buster
-* Support for using AS3 as an orchestration backend for Openshift Routes (Experimental)
+* Support for using AS3 as an orchestration backend for Openshift Routes
+* Added support for disabling Ingress resource processing using `--manage-partition` option
+* Master node will not be added as a pool member if marked as unscheduled in NodePort Mode
 
-Bug Fixes
-`````````
+Bugs Fixed
+``````````
 * Controller adds pods in unscheduled nodes as pool members.
```

Comment:
Duplicate of the above feature ?

Reply:
A bug is fixed, as part of that behavioural change occurred. So, it is mentioned at both places.

Reply:
Ok. Can you also provide the Issue raised in GITHUB for reference if any?

Reply:
It was a customer SR.